core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
+  <button class="btn btn-success" type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
```

Comment:
Can we use "btn btn-default" for this? I think we shouldn't have two green buttons; it diverts attention.